New build warning

```
formats.c: In function 'test_fmt_8_bit':
formats.c:735:21: warning: unused variable 'ret' [-Wunused-variable]
  char *plain_copy, *ret;
                     ^
```
@loverszhaokai is your compiler somehow not emitting this warning?

--- a/src/formats.c
+++ b/src/formats.c
@@ -732,7 +732,7 @@ static void test_fmt_8_bit(struct fmt_main *format, void *binary,
 	char *ciphertext, char *plaintext, int *is_ignore_8th_bit,
 	int *plaintext_is_blank)
 {
-	char *plain_copy, *ret;
+	char *plain_copy;
 
 	if (*plaintext == 0)
 		return;

